views/financial/income.pug

```diff
@@ -23,11 +23,11 @@ block content
         thead
           tr
             th.breakdown-table__heading #{__(`Total income earned in ${year}`)}
-            th.breakdown-table__heading $#{currencyFilter(totalIncome, 0)}
+            th.breakdown-table__heading $#{currencyFilter(totalIncome.amount, 0)}
         tbody
           tr
             th(colspan="2").breakdown-table__subheading #{__(`Detailed breakdown of income earned in ${year}`)}
-          each employer in income
+          each employer in incomeSources
```

Comment:
I suspect that these won't just be employers. It could be things like OAS, GIC, other provincial programs etc.